test/envtest/dlbdeviceplugin_controller_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2021 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package envtest
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	devicepluginv1 "github.com/intel/intel-device-plugins-for-kubernetes/pkg/apis/deviceplugin/v1"
+)
+
+var _ = Describe("DlbDevicePlugin Controller", func() {
+
+	const timeout = time.Second * 30
+	const interval = time.Second * 1
+
+	AfterEach(func() {
+		time.Sleep(time.Second * 2)
+
+	})
+
+	Context("Basic CRUD operations", func() {
+		It("should handle DlbDevicePlugin objects correctly", func() {
+			spec := devicepluginv1.DlbDevicePluginSpec{
+				Image: "testimage",
+			}
+
+			key := types.NamespacedName{
+				Name: "dlbdeviceplugin-test",
+			}
+
+			toCreate := &devicepluginv1.DlbDevicePlugin{
```

Comment:
what about to set namespace, Image and Args for container to cover this code? https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/pkg/controllers/dlb/controller.go#L78:L81